Classes/UVSuggestionSearchResultsController.m

```diff
@@ -0,0 +1,159 @@
+}
+
+- (void)customizeCellForResult:(UITableViewCell *)cell indexPath:(NSIndexPath *)indexPath {
+    [self customizeCellForSuggestion:[_searchResults objectAtIndex:indexPath.row] cell:cell];
+}
+
+- (void)initCellForSuggestion:(UITableViewCell *)cell indexPath:(NSIndexPath *)indexPath {
+    cell.backgroundColor = [UIColor whiteColor];
+    cell.accessoryType = UITableViewCellAccessoryDisclosureIndicator;
+    UIImageView *heart = [UVUtils imageViewWithImageNamed:@"uv_heart.png"];
+    UILabel *subs = [UILabel new];
+    subs.font = [UIFont systemFontOfSize:14];
+    subs.textColor = [UIColor grayColor];
+    subs.tag = SUBSCRIBER_COUNT;
+    UILabel *title = [UILabel new];
+    title.numberOfLines = 0;
+    title.tag = TITLE;
+    title.font = [UIFont systemFontOfSize:17];
+    UILabel *status = [UILabel new];
+    status.font = [UIFont systemFontOfSize:11];
+    status.tag = STATUS;
+    UIView *statusColor = [UIView new];
+    statusColor.tag = STATUS_COLOR;
+    CALayer *layer = [CALayer layer];
+    layer.frame = CGRectMake(0, 0, 9, 9);
+    [statusColor.layer addSublayer:layer];
+    NSArray *constraints = @[
+                             @"|-16-[title]-|",
+                             @"|-16-[heart(==9)]-3-[subs]-10-[statusColor(==9)]-5-[status]",
+                             @"V:|-12-[title]-6-[heart(==9)]",
+                             @"V:[title]-6-[statusColor(==9)]",
+                             @"V:[title]-4-[status]",
+                             @"V:[title]-2-[subs]"
+                             ];
+    [self configureView:cell.contentView
+               subviews:NSDictionaryOfVariableBindings(subs, title, heart, statusColor, status)
+            constraints:constraints
+         finalCondition:indexPath == nil
+        finalConstraint:@"V:[heart]-14-|"];
+}
+
+- (void)customizeCellForSuggestion:(UVSuggestion *)suggestion cell:(UITableViewCell *)cell {
+    UILabel *title = (UILabel *)[cell.contentView viewWithTag:TITLE];
+    UILabel *subs = (UILabel *)[cell.contentView viewWithTag:SUBSCRIBER_COUNT];
+    UILabel *status = (UILabel *)[cell.contentView viewWithTag:STATUS];
+    UIView *statusColor = [cell.contentView viewWithTag:STATUS_COLOR];
+    title.text = suggestion.title;
+    if ([UVSession currentSession].clientConfig.displaySuggestionsByRank) {
+        subs.text = suggestion.rankString;
+    } else {
+        subs.text = [NSString stringWithFormat:@"%d", (int)suggestion.subscriberCount];
+    }
+    [(CALayer *)statusColor.layer.sublayers.lastObject setBackgroundColor:suggestion.statusColor.CGColor];
+    status.textColor = suggestion.statusColor;
+    status.text = [suggestion.status uppercaseString];
+}
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+    tableView.backgroundView = nil;
+    tableView.separatorStyle = UITableViewCellSeparatorStyleSingleLine;
+    
+    if (_searchResults.count == 0) {
```

Comment:
This isn't really the right place to do this. I would think this controller should have like an `updateResults` method that could do this part, and also call `[self.tableView reloadData]`.